torch/_torch_docs.py

```diff
@@ -488,17 +488,17 @@ def parse_kwargs(desc):
 
 add_docstr(torch.atan2,
            r"""
-atan2(input1, input2, out=None) -> Tensor
+atan2(y_input, x_input, out=None) -> Tensor
```

Comment:
The function signature is actually atan2(input, other, out=None). This actually matters because one can use input and other as kwargs, i.e., `atan2(other=..., input=...)`. I think a more complete fix is to go in and change the signature of our atan2 function

Reply:
Hmm... Still, one has to specify which input is the ordinate and which the abscissa.
What is `other` supposed to mean?

Reply:
I do agree that `input`, `other` are bad names. I think we should change those names such that the code generation generates an atan2 function that has `y_input, x_input` arguments first before modifying the documentation.

Reply:
Almost got it. Could you point out what "code generation" refers to?

Reply:
Sorry @Atcold, I missed your comment. "code generation" refers to how we generate python bindings for our c/c++ backend functions. atan2 is defined [here](https://github.com/pytorch/pytorch/blob/fcb3ccf23fefe57525592a60c7091b0baf4b4e31/aten/src/ATen/Declarations.cwrap#L1812-L1828), with its arguments